vue-components/stories/Dialog.stories.ts

```diff
@@ -0,0 +1,83 @@
+import Dialog from '@/components/Dialog';
+import Button from '@/components/Button';
+import { Component } from 'vue';
+
+export default {
+	component: Dialog,
+	title: '/Dialog',
+};
+
+export function complex( args: { title: string, actions: string, dismissButton: boolean, visible: boolean, dialogText }): Component {
+	return {
+		components: { Button, Dialog },
+		props: Object.keys( args ),
+		template: `
+			<div>
+				<Button 
+					@click.native="$refs.simple.show()" 
+					variant="primary" type="progressive"
+				>
+					Open Basic Dialog
+				</Button>
+				<Dialog
+					:title="title"
+					ref="simple"
+					:actions="actions"
+					style="transform: translate(50%, 50%)"
+					@action="(_, dialog) => dialog.hide()"
+					:dismiss-button="dismissButton"
+					:visible="visible"
+				>
+					<p style="margin-bottom:10px">Hello, I am a simple dialog</p>
+					{{ dialogText }}
```

Comment:
tbh i am not sure about adding this here. it seems storybook doesnt accept html so it doesn't render optimally, if we want paragraphs and a title we need to add a separate control for each of them?

Reply:
I am not sure about this either. I definitely don't think we should add more than one control for the body of the dialog, but I'm not sure how valuable this is if we can't actually change the HTML. @sai-san what do you think?

Reply:
I agree that the controls are quite limited. Let's just include a single editable paragraph in the body, that's fine.

Reply:
oki :)